### PR TITLE
feature: CPA-289

### PR DESCRIPTION
**Pull request by itsmostafa**
Fix hanging Flower dashboard, add link to launch flower

## 🗣 Description
Fixed an issue where the flower dashboard was continuously loading. Added a button in admin to access the dashboard to view and manage scheduled tasks

## 💭 Motivation and Context
Flower dashboard needed to manually manage scheduled tasks

## 🧪 Testing
Successfully tested locally

## 🚥 Types of Changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by zenine07 — CHANGES_REQUESTED** [review]
Flower is not always going to be on the localhost. Can you add that url to the settings? If you need help doing that, let me know.

**Review by inlguy — APPROVED** [review]
in the user-admin component we are directly going to a localhost url.   It may not be localhost

File: controller/etc/entrypoint.sh [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 echo "Collecting static files"
 python manage.py collectstatic --no-input
-python scripts/init.py &
 
+echo "Initialize application"
+python scripts/init.py &
```
Comment (pmercado23) [inline controller/etc/entrypoint.sh:7]: this init script is my favorite thing lately 😄

**Review by pmercado23 — APPROVED** [review]
awesome, LGTM 🚢

**Review by zenine07 — APPROVED** [review]
changes good.

**Pull request merged**